### PR TITLE
Feat/iterative factorial

### PR DESCRIPTION
**Pull request by almacro**
I have some great iterative factorial code to merge.

### CODE REVIEW

**Comment by Derek-Underwood** [comment]
Gooooood

**Review by jorgefvillena — APPROVED** [review]
lgtm

**Comment by almacro** [comment]
I am not so sure about this - It doesn't work for floats!

File: main.rb [hunk 1]
```diff
@@ -1 +1,14 @@
-puts "Hello World!"
+n, rest_args = ARGV
+
+def factorial(n)
```
Comment (almacro) [inline main.rb:3]: Is this the right naming convention?

**Comment by almacro** [comment]
This is wrong. Merge to develop, not master branch.

**Pull request closed without merging**